src/components/schemas/fields.yaml

```diff
@@ -236,6 +236,149 @@ contains_operator:
     Value signifying the operator for the field
   example: IN
 
+expense_policy_rule_minimal:
+  type: object
+  properties:
+    id:
+      type: string
+      example: tpr1234abcd
+      description: This represents the id of the expense policy rule that is violated for this expense.
+    description:
+      anyOf:
+        - $ref: '#/description'
+      description: |
+        This represents the description of the expense policy rule that is violated for this expense.
+      example: Receipt is mandatory for expense above $50
+    action_show_warning:
+      type: boolean
+      description: |
+        A true value here means that policy calculation suggests showing a warning when the user attempts to create an expense that violates this expense policy rule.
+      example: True
+      
+expense_policy_violating_expenses_query_params:
+  type: object
+  description: |
+    An expense may be violating an expense policy rule due to a group of other expenses. 
+    These query params can be used to fetch those expenses because of which this expense is causing a violation. <br>
+    Below params should be ANDed to get the correct list of violating expenses.
+  properties:
```

Comment:
add conditions either in all or skip from all

Reply:
can try something like
[ {"param":"created_at", "operator":"gt", "value":"2022-11-11"},{"param":"created_at", "operator":"lt", "value":"2022-12-13"},{"param":"state", "operator":"neq", "value":"DRAFT"}]